package/docker/openmrs/Dockerfile

```diff
@@ -15,10 +15,17 @@ RUN curl -L -o /tmp/artifacts/liquibase-core-${LIQUIBASE_VERSION}.jar "https://o
 
 #unzip default_config
 ADD package/resources/default_config.zip /tmp/artifacts/
-RUN unzip -d /tmp/artifacts/default_config/ /tmp/artifacts/default_config.zip 
+RUN unzip -d /tmp/artifacts/default_config/ /tmp/artifacts/default_config.zip
 
 COPY distro/target/distro/*.omod /usr/local/tomcat/.OpenMRS/modules/
 COPY package/docker/openmrs/templates/bahmnicore.properties.template /etc/bahmni-emr/templates/
+
+# Add the latest ot, fhir and web-services modules with OMRS:2.4.2 changes
```

Comment:
I hope the reason for adding the OMODS here is because these are not yet released with the required changes. And this would be removed as soon as the artefacts are released right ??